var/spack/repos/builtin/packages/hicops/package.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Hicops(Package):
```

Comment:
You can derive from a specialized class to use a specific build-system:
```suggestion
class Hicops(CMakePackage):
```

Reply:
Hi,
Just a minor comment but the 'Hicops' package name can stay in lowercase like in your original commit for consistency with other spack packages.

Reply:
@mhaseeb123 it has to be CapWords convention to be pep8 compliant and pass the styling check within this repo.

Reply:
@NessieCanCode thank you for the clarification. As long as the package name remains all lowercased, all is well.